docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[ml-nlp-deploy-models]]
+= Import and deploy trained models
+
+:keywords: {ml-init}, {stack}, {nlp}
+:description:  To make trained models available for inference, you must import \
+and deploy them in {es}.
+
+If you want to perform {nlp} tasks in your cluster, you must import and deploy
+an appropriate trained model. There are {es} APIs and tooling support in eland
+to prepare and manage models. 
+
+. <<ml-nlp-select-model,Choose a trained model.>>
+. <<ml-nlp-upload-model,Upload the trained model and vocabulary.>>
+. <<ml-nlp-deploy-model,Deploy the model in your cluster.>>
+
+[discrete]
+[[ml-nlp-select-model]]
+== Select a trained model
+
+Per <<ml-nlp-overview>>, there are multiple ways that you can use NLP features
+within the {stack}. After you determine which type of NLP task you want to
+perform, you must choose an appropriate trained model. You can either create the
+model yourself or find one that meets your needs.
+
+IMPORTANT: The {stack-ml-features} support transformer models that conform to
+the standard BERT model interface and use the WordPiece tokenization algorithm.
+
+The simplest method is to use a model that has already been fine-tuned for the
+type of analysis that you want to perform. For example, there are models and 
+data sets available for specific NLP tasks in
+https://huggingface.co/models[Hugging Face].
+
+[discrete]
+[[ml-nlp-upload-model]]
+== Upload the trained model and vocabulary
+
+After you find or create a model, you must upload it and its tokenizer
+vocabulary to your cluster. 
+
+Trained models must be in a TorchScript representation for use with
+{stack-ml-features}. To use scripts that format and upload the necessary
+artifacts, refer to https://github.com/elastic/eland#nlp-with-pytorch
+
+Alternatively, you can use the
+{ref}/put-trained-model-vocabulary.html[create trained model vocabulary API] and
+{ref}/put-trained-models.html[create trained models API] or
+{ref}/put-trained-model-definition-part.html[create trained model definition part API].
+// When you upload the model, it must be chunked and uploaded one chunk at a time. 
+//TBD Why? How?
```

Comment:
This is true, that's why I'm not sure we should even mention the API's directly but just the eland scripts since they encapsulate the chunking, for example. We have to chunk the model since it is stored in parts in an ES index, and we can't store very large (1GB+) models as a single document.

Reply:
Thanks, for the details!  For now, I've qualified that paragraph with the recommendation to use eland.